.overcommit.yml

```diff
@@ -0,0 +1,49 @@
+# Use this file to configure the Overcommit hooks you wish to use. This will
+# extend the default configuration defined in:
+# https://github.com/brigade/overcommit/blob/master/config/default.yml
+#
+# At the topmost level of this YAML file is a key representing type of hook
+# being run (e.g. pre-commit, commit-msg, etc.). Within each type you can
+# customize each hook, such as whether to only run it on certain files (via
+# `include`), whether to only display output if it fails (via `quiet`), etc.
+#
+# For a complete list of hooks, see:
+# https://github.com/brigade/overcommit/tree/master/lib/overcommit/hook
+#
+# For a complete list of options that you can use to customize hooks, see:
+# https://github.com/brigade/overcommit#configuration
+#
+# Uncomment the following lines to make the configuration take effect.
```

Comment:
nit: I general I prefer not to have have things checked in and commented out, including instructions like this and defaults like at the end of the file.  Similar to the way we don't have the default config/routes.rb comments/instructions checked int.

Otherwise LGTM.  I don't feel too strongly about this, feel free to merge either way.

Not a big